main.go

```diff
@@ -164,7 +167,38 @@ func (s *admissionWebhookServer) createVolumesPatch(p string, volumes []corev1.V
 	return jsonpatch.NewOperation("add", path.Join(p, "volumes"), volumes)
 }
 
+func parseInterfacePools(v string, logger *zap.SugaredLogger) map[string]int {
```

Comment:
Probably this one would be better?
```suggestion
func parseSRIOVLabels(v string, logger *zap.SugaredLogger) map[string]int {
```

Reply:
ok I will fix that. Thanks!

Reply:
@Bolodya1997 Done